.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+/.idea/
```

Comment:
Did you mean to commit this?

Reply:
Yes. The .idea folder contains mostly internal Intellij config files that I would rather not commit. Adding it to .gitignore makes sure that I (or anyone in the future) won't commit these files by accident.

Reply:
You can keep changes like this local by putting them into a changeset separate from the one you're committing. This way stuff specific to your setup doesn't bleed into the main repository.